DependencyInjection/Configuration.php

```diff
@@ -17,6 +17,7 @@
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 use Symfony\Component\Config\Definition\ConfigurationInterface;
 use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+use Twig\Extension;
```

Comment:
This class does not exists. I suggest to either use `Twig\Environment` as suggested by @jackbakker-hostnet or `AbstractExtension` because this is the replacement of `\Twig_Environment`